stylesheet/style.css

```diff
@@ -0,0 +1,6 @@
+.header {
+    color: purple;
+    text-align: center;
+    font-size: 62px;
+    font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
![third hello microverse linters](https://user-images.githubusercontent.com/93039359/176232626-60cf5545-752f-4583-bcfc-ebde16a49ca7.png)

-  [ ] It's great that you have implemented the linters correctly :+1: But your linters are not passing all the tests. Please check your linter errors and kindly try to fix them :innocent:

Reply:
@HAMMAS-SALEEM the last error on linters . I have tried every possible way to fix it but still gives the same error. Though I have fixed the other pending issues.
Here is the error;
Unexpected missing end-of-source newline   no-missing-end-of-source-newline